src/components/BookCollection/BookCollection.css

```diff
@@ -1,9 +1,15 @@
+.book-collection-main-div {
```

Comment:
Try following same naming convention for css classes, is it -div or -box suffix? When do you embed name of the component, ('book-collection') in the css and when you leave it out?